CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+
+### Discuss the contribution with the SKY UX team
+
+After you file an issue, discuss the architecture and implementation details, requirements, and timelines with a SKY UX engineer. We will assign a developer to your issue to help answer questions.
+
+### Developer setup
+
+1. Fork the repo into your own GitHub repo.
+1. Create a branch off of `main`. Name it after the feature to contribute, and use kebab-case (`my-new-feature`).
+1. Clone your repo locally and run `nvm use` to ensure you have the preferred version of Node.js installed. Then run `npm install` to install all required dependencies. For more information on prerequisites, see [SKY UX prerequisites](https://developer.blackbaud.com/skyux/learn/get-started).
+
+### Request a preliminary review
+
+Create a pull request with the initial implementation, functionality, and styling for an initial review. Make changes and request additional reviews as necessary. For [large changes that effect more than 50 files](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#pull-requests), submit changes in multiple small pull requests instead of one large pull request.
+
+Wait until after the initial review to add tests and documentation because implementation details may change.
+
+### Add tests
+
+Write tests for your feature. All code requires 100 percent [unit test](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#unit-tests) code coverage, and components also require [visual regression tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#visual-regression-tests) and [accessibility tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#accessibility-tests). In addition, create [test fixtures](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#test-fixtures) for any new components.
+
+### Request a final review
+
+1. Before requesting a final review, [squash all commits on your feature branch into one](#squashing-all-commits-into-one). This will consolidate all your work and include a well-formed commit message to be used in the changelog.
+1. Reach out to a SKY UX engineer to review the complete contribution. We review the implementation, testing, functionality, styling, localization, and accessibility.
+
+#### Address review feedback
+
+1. Make required changes to the source code.
+1. Rerun all test suites.
+1. Create a "fixup" commit:
+
+```
+git commit --all --fixup HEAD
+git push
+```
+
+#### Update the commit message
+
+If the reviewer asks you to reword your commit message, use the `--amend` flag:
+
+```
+git commit --amend
+git push --force-with-lease
+```
+
+_Note: To amend a previous commit, you need to [squash all commits into one](#squashing-all-commits-into-one)._
```

Comment:
```suggestion
_Note: To amend a previous commit, you need to [squash all commits into one](#squash-all-commits-into-one)._
```
I assume we need to tweak this link to reflect the change to the header. Is that right? (Of course, given that it's linking to the next line in the file, maybe the link isn't really necessary.)